### PR TITLE
Coding standards

### PR DESCRIPTION
**Pull request by rossabaker**
We've been talking a lot about making the model more consistent.  Here's a start on explaining it.

### CODE REVIEW

File: docs/src/hugo/content/standards.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+## Types
+
+### Effects
+
+Prefer a parameterized effect type over specializing on a task.
+
+```scala
+// Good
+def apply[F[_]](service: HttpService[F])(implicit F: Monad[F]): HttpService[F]
```
Comment (ChristopherDavenport) [inline docs/src/hugo/content/standards.md]: Doesnt this apply to 0.18 or after effect is more presently, we are hard-coded to `Task` .

**Comment by aeons** [comment]
Would formatting fit here as well? It's kinda all over the place currently.

[scalafmt](http://scalafmt.org) with a sane config might be interesting. We use that all the time.

File: docs/src/hugo/content/standards.md (hunk 1 above)
Comment (rossabaker) [inline docs/src/hugo/content/standards.md]: True, this section is forward looking.  I'll add something about that.

**Comment by rossabaker** [comment]
I would like to start using scalafmt, but I'm scared to with the branches in flux.  I think we should strongly consider introducing scalafmt when the fs2-0.10 is the main line of development.

**Comment by rossabaker** [comment]
`parse` or `fromString`?  We're all over the place here, too.

**Comment by aeons** [comment]
`fromString` is more in line with the other methods imo.

**Pull request merged**